make/Makefile.test

```diff
@@ -96,18 +96,27 @@ $(GINKGO):
 
 
 #
-# ------------ test for raw images  ------------
+# ------------ test for raw image  ------------
 #
 
 RAW?=$(shell ls $(ROOT_DIR)/*.raw 2> /dev/null)
 
-raw_disk_test: as_root $(VBOXMANAGE)
+raw_disk_test_deploy: create_vm_from_raw_image $(GINKGO)
+	# run tests
+	cd $(ROOT_DIR)/tests && $(GINKGO) $(GINKGO_ARGS) ./recovery-raw-disk
+
+create_vm_from_raw_image: as_root $(VBOXMANAGE)
 ifeq ("$(RAW)","")
 	@echo "Raw image does not exists, please run make raw_disk first"
 	@exit 1
 else
 	# transform the raw image
 	VBoxManage convertdd $(RAW) sda.vdi --format VDI
+	# increase disk size to 15Gb
+	VBoxManage modifymedium disk sda.vdi --compact --resize 16000
```

Comment:
that bugs me a bit, aren't we going to test an image which is a bit larger than the one we provide?

Reply:
We are resizing the image otherwise the user-data cannot partition it correctly (lack of space) this is to be on sync with what we do on cloud operators (resize image on launch)